deployment/terraform/task-definitions/transit.json

```diff
@@ -10,6 +10,14 @@
                 "containerPort": 9999,
                 "hostPort": 0
             }
-        ]
+        ],
+        "logConfiguration": {
+            "logDriver": "awslogs",
+            "options": {
+                "awslogs-group": "log${transit_environment}ContainerInstance",
```

Comment:
I think that it would be useful to rename this log group such that it is specific to the service. If the service has multiple container definitions, we can use the stream prefix setting to differentiate.